src/include/sof/schedule.h

```diff
@@ -55,6 +56,8 @@ struct sof;
 #define TASK_PRI_MED	0
 #define TASK_PRI_HIGH	-20
 
+/* maximun task time slice in miliseconds */
+#define SCHEDULE_TASK_MAX_TIME_SLICE	5000
```

Comment:
The comment says milliseconds, but time is in micro seconds.

Reply:
yes, correct. I think 50us is too short, so I change it to 5ms.